### DISCUSSION TITLE
Support for factory API's & /status output

### DISCUSSION THREAD

**Discussion by jhardydev** [mongoose-os-apps/shelly-homekit#1482] [Ideas]
Some of the home automation tools (like Control4) require the factory API's & the /status output from the Shelly module.  In order to support both the HomeKit integration and integration with Control4, as well as the VERY nice to have feature where the external switch will always implement to opposite relay state (open to close or close to open) regardless of the position the switch is in, it would be great to have the API's & /status functionality ported over.

Thank you!

**Comment by rojer** [comment]
sounds good, no objections as long as producing the output does not require major new functionality.
